Twittimation/Commands/InteractiveMode.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Twittimation.Commands
+{
+    public sealed class InteractiveMode : Command
+    {
+        public override List<string> RequiredArgs { get; } = new List<string>();
+        public override List<string> OptionalArgs { get; } = new List<string>();
+        public override Optional<string> OptionalRepeatedArg { get; } = new Optional<string>();
+        public override string HelpInfo { get; } = "Allows you to use any number of commands without repeatedly opening the program.";
+        public override string ExtendedHelp => HelpInfo;
+
+        private Cli _cli;
+
+        public InteractiveMode(Cli cli)
+        {
+            _cli = cli;
+        }
+
+        protected override void Go(string[] args)
+        {
+            var commands = _cli.Commands;
+            commands.Remove(GetType().Name);
```

Comment:
It's a very dangerous practice to mutate the content of an object that has been injected into multiple classes. If you need different commands, it's better to inject a different instance, instead of mutating.